Source/Workers/createPointGeometry.js

```diff
@@ -0,0 +1,6 @@
+/*global define*/
+define(['../Core/PointGeometry'], function(PointGeometry) {
```

Comment:
Sorry, what I meant was, merge master down into this branch and then run sortRequires.  This way you know there aren't any jsHint warnings too.